submit_and_compare/submit_and_compare.py

```diff
@@ -183,26 +191,28 @@ def studio_view(self, context=None):
             'our_answer_label': self.our_answer_label,
             'submit_button_label': self.submit_button_label,
         }
-        html = self.render_template(
+        html = SubmitAndCompareXBlock.render_template(
```

Comment:
Since render_template is not used outside of this file, it could be marked as private/protected by prefixing the name with an underscore.